Sources/NIO/ByteBuffer-core.swift

```diff
@@ -708,6 +708,7 @@ public struct ByteBuffer {
     ///
     /// - parameters:
     ///     - minimumCapacity: The minimum capacity that will be (re)allocated for this buffer
+    @available(*, deprecated, message: "Use an `Int` as the argument")
     public mutating func clear(minimumCapacity: _Capacity) {
```

Comment:
Can we switch `_Capacity` to whatever its type alias is currently is?

Reply:
I can do - wasn't sure if this is technically an API change, @Lukasa any opinions?

Reply:
It's not, typealiases are exactly the same as their underlying type. It only matters if you remove the name altogether, which we aren't.